src/ansible/roles/packages/tasks/RedHat8.yml

```diff
@@ -0,0 +1,6 @@
+- name: Enable idm module
```

Comment:
I still think you want to include CentOS8.yml here. You limit the packages that are installed with "extended_packageset". If that is set to True then EPEL and sssd-ci-deps repo will be missing.

Reply:
Added epel for extended packageset. We should be god now. 
The ci-deps will be in place with https://github.com/SSSD/sssd-ci-containers/pull/51